slurm-spank-lua.spec

```diff
@@ -11,9 +11,9 @@ Group: System Environment/Base
 Source0: %{name}-%{version}.tar.gz
 BuildRoot:  %{_tmppath}/%{name}-%{version}
 
-BuildRequires: slurm-devel bison flex
-BuildRequires: lua-devel >= 5.1
-Requires:      slurm lua-devel >= 5.1
+BuildRequires: slurm-devel bison flex pkg-config
```

Comment:
Ah, unfortunately, the RHEL/CentOS package is pkgconfig, not pkg-config.

Both RHEL and SuSE packages seem to provide pkgconfig(pkg-config) though, so maybe we can use this instead?

Reply:
Ah, sorry. I matched what Slurm already used (https://github.com/SchedMD/slurm/blob/5e6962316781c5faeccb7e73b74921a085d42885/slurm.spec#L107) but now see that's inside an `%if`